src/components/10-atoms/link/index.js

```diff
@@ -1,3 +1,6 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import '@axa-ch/icon/lib/index';
```

Comment:
i think `@axa-ch/link` is enough

Reply:
Ah, makes sense, because of the main property. Thx :)